py/test/selenium/selenium_test_suite.py

```diff
@@ -29,6 +30,7 @@ def suite():
         unittest.makeSuite(test_default_server.TestDefaultServer),
         unittest.makeSuite(test_google.TestGoogle),
         unittest.makeSuite(test_i18n.TestI18n),
+        unittest.makeSuite(test_remote_connection.TestRemoteConnection),
```

Comment:
@lukeis Cool thanks! 
@davehunt @AutomatedTester Please help, will make the changes accordingly.